data_steward/cdr_cleaner/cleaning_rules/truncate_fitbit_data.py

```diff
@@ -0,0 +1,213 @@
+"""
+Remove all FitBit data newer than the cutoff date for participants
+
+Original Issue: DC-1046
+
+The intent is to ensure there is no data newer than cutoff date for participants in
+Activity Summary, Heart Rate Minute Level, Heart Rate Summary, and Steps Intraday tables
+by sandboxing the applicable records and then dropping them.
+"""
+
+# Python Imports
+import logging
+
+# Third Party Imports
+from jinja2 import Environment
+
+# Project Imports
+import common
+import constants.cdr_cleaner.clean_cdr as cdr_consts
+from cdr_cleaner.cleaning_rules.base_cleaning_rule import BaseCleaningRule
+from constants.bq_utils import WRITE_TRUNCATE
+
+LOGGER = logging.getLogger(__name__)
+
+jinja_env = Environment(
+    # help protect against cross-site scripting vulnerabilities
+    autoescape=True,
+    # block tags on their own lines
+    # will not cause extra white space
+    trim_blocks=True,
+    lstrip_blocks=True,
+    # syntax highlighting should be better
+    # with these comment delimiters
+    comment_start_string='--',
+    comment_end_string=' --')
+
```

Comment:
Instead of writing this out here, you can use the environment declared in`utils.bq.py` and remove lines 25-35: 

`from utils.bq import JINJA_ENV`

Then on lines 53 and 60 it would be `JINJA_ENV.from_string(`